core/lib/object_store/src/objects.rs

```diff
@@ -92,7 +94,12 @@ impl StoredObject for SnapshotFactoryDependencies {
         decoder
             .read_to_end(&mut decompressed_bytes)
             .map_err(BoxedError::from)?;
-        serde_json::from_slice(&decompressed_bytes).map_err(From::from)
+        let proto =
+            <SnapshotFactoryDependencies as ProtoFmt>::Proto::decode(&decompressed_bytes[..])
+                .context("decode SnapshotFactoryDependencies")?;
+        SnapshotFactoryDependencies::read(&proto)
+            .context("deserialization of Message to SnapshotFactoryDependencies")
+            .map_err(From::from)
```

Comment:
Nit: Please use `zksync_protobuf::decode` here as well for consistency.

Reply:
yes, sorry for not noticing this, corrected